addons/product_expiry/tests/test_generate_serial_numbers.py

```diff
@@ -163,6 +163,49 @@ def test_set_multiple_lot_name_with_expiration_date_04_written_months(self):
             self.assertEqual(move_line.quantity, 1)
             self.assertEqual(move_line.expiration_date, list_lot_and_qty[i]["datetime"])
 
+    @freeze_time('2025-9-13')
+    def test_set_multiple_lot_name_with_expiration_date_05_import_dates(self):
+        """ When importing lot names, quantities and expiration dates, make sure the date is not
+        replaced by the computed date and if there's no date, it takes the computed date. """
+        product_lot = self.env['product.product'].create({
+            'name': 'Tracked by Lot Numbers',
+            'tracking': 'lot',
+            'is_storable': True,
+            'use_expiration_date': True,
+            'expiration_time': 10,
+        })
+        receipt_picking = self.env['stock.picking'].create({
+            'picking_type_id': self.warehouse.in_type_id.id,
+            'location_id': self.env.ref('stock.stock_location_suppliers').id,
+            'location_dest_id': self.warehouse.lot_stock_id.id,
+            'state': 'draft',
+        })
+        self.env['stock.move'].create({
+            'product_id': product_lot.id,
+            'product_uom': product_lot.uom_id.id,
+            'product_uom_qty': 20,
+            'picking_id': receipt_picking.id,
+            'location_id': receipt_picking.location_id.id,
+            'location_dest_id': receipt_picking.location_dest_id.id,
+        })
```

Comment:
```suggestion
        receipt_picking = self.env['stock.picking'].create({
            'picking_type_id': self.warehouse.in_type_id.id,
            'location_id': self.env.ref('stock.stock_location_suppliers').id,
            'location_dest_id': self.warehouse.lot_stock_id.id,
            'move_ids': [Command.create({
                'product_id': product_lot.id,
                'product_uom_qty': 20,
                'location_id': self.env.ref('stock.stock_location_suppliers').id,
                'location_dest_id': self.warehouse.lot_stock_id.id,
             })],
        })
```
Single create, and no need to give unecessary vals